Block/Adminhtml/System/Config/Form/YotpoButton.php

```diff
@@ -51,7 +51,7 @@ protected function _getElementHtml(\Magento\Framework\Data\Form\Element\Abstract
      * @return string
      */
     public function getAjaxExportUrl()     { 
-        return '/admin/massmap/yotpocontroller/yotpocontroller/';
+        return $this->getUrl('massmap/yotpocontroller/yotpocontroller/', ['_secure' => $this->getRequest()->isSecure()]);
```

Comment:
Check out https://magento2.atlassian.net/wiki/display/m1wiki/getUrl%28%29+Function+Parameters
tl;dr: the first param is module/controller/action, second params is options, specifically I want to keep the same 'secure' behaviour I accepted in the request.